.github/workflows/staging-deploy.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Staging Deploy
+
+on:
+  push:
+    branches:
+    - develop
+
+env:
+  PROJECT_ID: ${{ secrets.GCE_PROJECT }}
+
+jobs:
+  setup-build-publish-deploy:
+    name: Setup, Build, Publish, and Deploy
+    runs-on: ubuntu-latest
+    environment:
+      name: staging
+      url: 'http://35.245.13.116:4000'
```

Comment:
Why have you use maha staging IP address: 35.245.13.116 and not haha staging IP address: 34.150.173.73?

Reply:
we will change that after we get a static IP for haha, it doesn't affect updating the instance

Reply:
Let it be changed before merging, we might forget about it